src/views/pages/name/private-limited-partnership-name-scotland.njk

```diff
@@ -0,0 +1,104 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form class="form" action={{ props.currentUrl }} method="post">
+
+      <input type="hidden" name="pageType" value={{ props.pageType }}>
+
+      {% include "includes/csrf_token.njk" %}
+
+      <h1 class="govuk-heading-xl">{{ i18n.privateNameScottishPage.title }}</h1>
+
+      <div class="govuk-form-group">
+        <h1 class="govuk-label-wrapper">
+          <label class="govuk-label govuk-label--l" for="partnership_name">
+            {{ i18n.privateNameScottishPage.whatIsName }}
+          </label>
+        </h1>
+
+        {{ govukInput({
+          classes: "govuk-input--width-40",
+          errorMessage: errors.private_partnership_scotland_name if errors,
+          id: "partnership_name",
+          name: "partnership_name",
+          value: partnership_name,
```

Comment:
the value is supposed to be props.data.limitedPartnership.partnership_name

Reply:
Confused. It's this value in all the other page templates and code seems to work. Do you think it needs changing?

Reply:
yes, it needs changing
because we don't use real values for the moment, need to create get endpoint

Reply:
Done.